backend/spec/system/backoffice/investors_spec.rb

```diff
@@ -130,12 +162,30 @@
     context "account language section" do
       before { within_sidebar { click_on t("backoffice.account.account_language") } }
 
-      it "can update account language" do
-        expect(page).to have_select(t("simple_form.labels.account.language"), selected: "English")
-        select "Spanish", from: t("simple_form.labels.account.language")
-        click_on t("backoffice.common.save")
-        expect(page).to have_text(t("backoffice.messages.success_update", model: t("backoffice.common.investor")))
-        expect(approved_investor.account.reload.language).to eq("es")
+      context "when new language has all text translated" do
+        before do
+          approved_investor.account.update! about_es: "ES"
+          approved_investor.update! other_information_es: "ES", mission_es: "ES", prioritized_projects_description: "ES"
+        end
+
+        it "can update account language" do
+          expect(page).to have_select(t("simple_form.labels.account.language"), selected: "English")
+          select "Spanish", from: t("simple_form.labels.account.language")
+          click_on t("backoffice.common.save")
+          expect(page).to have_text(t("backoffice.messages.success_update", model: t("backoffice.common.investor")))
+          expect(approved_investor.account.reload.language).to eq("es")
+        end
+      end
+
+      context "when new language is missing some translations" do
```

Comment:
Great to have it and those missing errors in the form in case someone will do that.